index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+
+    <!-- Font awesome link -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+      crossorigin="anonymous"
+    />
+    <title>COP27</title>
+  </head>
+  <body>
+    <header>
+
+      <!-- Contact links from font awesome -->
+      <div id="secondary-nav-bar" class="container">
+        <div class="wrapper">
+          <a href="#"><i class="fa fa-mastodon"></i></a>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+
+      <!-- Navbar with sub-titles link -->
+      <div id="nav-bar">
+        <a href="#" id="logo">COP27<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
```

Comment:
- [ ] please your hamburger icon does not work on the about page of the mobile version. please kindly check that
- [ ] kindly ensure that background of your menu is transparent or the same color/pattern as the top section background.

Reply:
fixed :+1: